GoogleForCreators/web-stories-wp
# Redesign: Color Input

## Context



## Summary
Redesign for color input.

- Moves hex input to design-system
- Uses the hex input for color input in the editor
- Updates the style for the color input


## Relevant Technical Choices
Did not move the full color input to the design-system since it has a lot of dependencies including the color picker lib and a bunch of general utils.

We could try to move everything except for the color picker to design-system, assuming that the color input could use any color picker solution and not a fixed one. Doesn't seem urgent now though.

Hex is a separate input and not a variant of `Input` -- should we change it?


## To-do

- [x] See that tests still work
- [x] Should hex be a variant of input?



## User-facing changes
| Before  | After |
| ------------- | ------------- |
| <img width="262" alt="Screenshot 2021-02-23 at 17 59 15" src="https://user-images.githubusercontent.com/3294597/108870543-80500880-75f8-11eb-854a-0cd99949b8fc.png"> | <img width="302" alt="Screenshot 2021-02-23 at 17 57 37" src="https://user-images.githubusercontent.com/3294597/108870313-4e3ea680-75f8-11eb-814f-8adc16f2c686.png">  |
| <img width="276" alt="Screenshot 2021-02-23 at 17 59 07" src="https://user-images.githubusercontent.com/3294597/108870577-89d97080-75f8-11eb-8d75-054116990c44.png"> | <img width="296" alt="Screenshot 2021-02-23 at 17 57 45" src="https://user-images.githubusercontent.com/3294597/108870371-5991d200-75f8-11eb-92e6-4c7f4036b4e3.png"> |
| - | <img width="397" alt="Screenshot 2021-02-23 at 18 01 27" src="https://user-images.githubusercontent.com/3294597/108870828-ca38ee80-75f8-11eb-8496-c8f1e3b6be4f.png"> |


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add an element that uses colors (e.g. a shape or text)
2. Verify that the color picker looks like in Figma

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6456
